opentelemetry-exporter-geneva/geneva-uploader/src/payload_encoder/otlp_encoder.rs

```diff
@@ -280,6 +281,9 @@ impl OtlpEncoder {
         };
 
         // Create single batch with all spans
+        let schemas_count = schemas.len();
+        let events_count = events.len();
```

Comment:
nit: It looks like you might have unintentionally moved these two lines under this comment.